dockergen/nginx.tmpl

```diff
@@ -71,6 +72,7 @@ server {
 {{ range $url, $containers := groupByMulti $containersByHost "Env.VIRTUAL_URL" "," }}
     location {{ $url }} {
         proxy_pass http://{{ $host }}{{ replace $url "/" "-" -1 }}u/;
+        proxy_redirect https://{{ $host }}{{ replace $url "/" "-" -1 }}u/ /;
```

Comment:
`u` at the end should not be exposed to the container? So not sure where this is coming from.

Will it work without `https` for you? If your container is redirecting to `https` version, then you might want to redirect further with `https` as well? So have regex match the protocol?